Branch-SDK/Branch-SDK/BranchShortUrlRequest.m

```diff
@@ -62,7 +63,7 @@ - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
         if (self.callback) {
             NSString *failedUrl = nil;
             NSString *userUrl = [BNCPreferenceHelper preferenceHelper].userUrl;
-            if (!userUrl) {
+            if (userUrl) {
```

Comment:
whoa, so it used to pass nil to the createLongUrlForUserUrl function without any problem?